src/test/java/showcase/neodymium/tests/logging/LoggingSecondTest.java

```diff
@@ -0,0 +1,27 @@
+package showcase.neodymium.tests.logging;
+
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.junit.Test;
+
+import io.qameta.allure.Description;
+
+public class LoggingSecondTest
+{
+    private static Logger logger = LogManager.getLogger(LoggingSecondTest.class);
+
+    @Test
```

Comment:
add a description to explain what is different compared to the first test

Reply:
There is no difference. is just a second test class for a second log file.

Reply:
Thats a difference and the reason for the second test. Please add this information as a comment.

Reply:
done